.eslintrc.yml

```diff
@@ -73,9 +72,6 @@ rules:
   ###########
   # SPECIAL #
   ###########
-  prettier/prettier:
```

Comment:
since we always run `prettier` and `eslint` together, this meant that we were effectively linting formatting twice every time

Reply:
LGTM 👍